### DISCUSSION TITLE
Why mlforecast.predict does not handle NA values?

### DISCUSSION THREAD

**Discussion by Ehsan86** [Nixtla/mlforecast#373] [Q&A]
To just give a bit context here, I always receive data with a 3-day delay. This means that at time `T`. I have data only from `T-4` back to `T-100`. Therefore, I do not have access to data for `T-1`, `T-2`, and `T-3`. Given that, I want to train a model based on data from `T-100` to `T-3` and then test it for time `T`. How can MLForecast handle this? When I use `mlforecast.predict` it gives me an error indicating that the data is missing, referring to the missing data from `T-1` to `T-3.` Can anyone help me with this issue as my train data and test are not continuous?

**Comment by jmoralez** [comment]
Same as #371
